src/osha/oira/client/browser/report.py

```diff
@@ -201,11 +201,15 @@ def create_workbook(self):
                         ):
                             description = zodb_node[measure.solution_id].description
                             value = f"{description}\n{value}"
-                        value = portal_transforms.convertToData("text/plain", value)
+                        value = portal_transforms.convertToData(
+                            "text/plain", value
+                        ).strip()
```

Comment:
This method can also return `None`:
https://github.com/plone/Products.PortalTransforms/blob/c3bd4a1874fbc8b61622e8aaae94d8c712adbfa2/Products/PortalTransforms/TransformEngine.py#L200-L217

I would be more defensive, I let you decide how.

Reply:
Fair point. I've added a condition.